index.js

```diff
@@ -162,6 +164,58 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.split(" ").splice(1).join(" ");
```

Comment:
difference between `.slice()` and `.splice()` is very confusing, but basically `splice` changes the original array, whereas `slice` creates a new array. We generally want to avoid _mutating_ values (same reason we prefer `const` over `let`), so best to use `slice` instead of `splice`. Luckily, they're interchangeable.

E.g.:
```js
const arr1 = [3, 5, 6];
const arr2 = arr1.slice(1);

console.log(arr1); // 3, 5, 6
console.log(arr2); // 5, 6
```

vs

```js
const arr1 = [3, 5, 6];
const arr2 = arr1.splice(1);

console.log(arr1); // 5, 6
console.log(arr2); // 5, 6
```